constraint_utils/constraint_basics.py

```diff
@@ -800,6 +848,19 @@ def old_new_compare(source, axis_type='log'):
 
 
 def pLI(ht: hl.Table, obs: hl.expr.Int32Expression, exp: hl.expr.Float32Expression) -> hl.Table:
+    """
+    Compute pLI score using the number of observed and expected variants.
+    
+    The output Table will include the following annotations:
+        - pLI
+        - pNull
+        - pRec
+
+    :param ht: Input Table.
+    :param obs: The number of observed variants on each gene or transcript.
+    :param exp: The number of expected variants on each gene or transcript.
```

Comment:
These are actually expressions
```suggestion
    :param obs: Expression for the number of observed variants on each gene or transcript in `ht`.
    :param exp: Expression for the number of expected variants on each gene or transcript in `ht`.
```